library/core/src/slice/mod.rs

```diff
@@ -993,6 +993,36 @@ impl<T> [T] {
         ChunksExactMut { v: fst, rem: snd, chunk_size }
     }
 
+    /// Splits the slice into a slice of `N`-element arrays,
+    /// starting at the beginning of the slice,
+    /// and a remainder slice with length strictly less than `N`.
+    ///
+    /// # Panics
+    ///
+    /// Panics if `N` is 0. This check will most probably get changed to a compile time
+    /// error before this method gets stabilized.
```

Comment:
"Will most probably"? Does it not panic at compile time?
```suggestion
    /// Panics if `N` is 0 at compile time.
```

Reply:
This is unstable, and the idea is that the behaviour will change before stabilization.  This is the same comment as on the `array_chunks(_mut)` methods.

Reply:
(I suppose the other possibility would be to put everything in the remainder for N=0, but that would require weakening the `remainder.len() < N` postcondition, which I'd rather not do.)

Reply:
I'm probably missing something: what language feature would allow us to make the 0 case a compile time error? A `where N > 0` bound?

Reply:
Yeah, or a trait that's only implemented for `[T; N+1]` or something, implicitly causing that restriction.

This is copied from the same text in https://doc.rust-lang.org/nightly/std/primitive.slice.html#method.array_chunks